foundry.toml

```diff
@@ -36,6 +36,7 @@ additional_compiler_profiles = [
   { name = "v3-periphery-default", optimizer_runs = 1000000, evm_version = "istanbul", via_ir = false },
   { name = "permit2", optimizer_runs = 1000000, via_ir = true },
   { name = "uniswapx", optimizer_runs = 1000000, via_ir = false },
+  { name = "uerc20-factory", optimizer_runs = 50000000, via_ir = false }
```

Comment:
@gretzke i think this line still isn't working - i deployed on unichain sepolia and sepolia and it's still being deployed with 200 runs :(

Reply:
https://sepolia.etherscan.io/address/0xf3e6e8dB96Bd434CC633586E008A23ED1283Ef62#code

https://sepolia.uniscan.xyz/address/0x40c6D986E4d4BA38E64E22Da10991105Ed251996#code